types/types.d.ts

```diff
@@ -1425,6 +1425,39 @@ export interface Page {
     timeout?: number;
   }): Promise<void>;
 
+  /**
+   * @param selector1
+   * @param selector2
+   * @param options
+   */
+  dragAndDrop(selector1: string, selector2: string, options?: {
```

Comment:
Let's call it maybe source and target everywhere? So it also reflects nicer in auto completion. Some editors show the parameter name in the parameter list and selector1 and selector2 might be not idiomatic.

Reply:
done.